"from psutil import *" throws AttributeError

Running RHEL 6.4, Python 2.6.6 against psutil from master

```
wjia@wjia-ld1 ~/git/salt (git)-[psutil_compat] % pip install git+https://github.com/giampaolo/psutil.git
Downloading/unpacking git+https://github.com/giampaolo/psutil.git
  Cloning https://github.com/giampaolo/psutil.git to /tmp/pip-OAeEcD-build
  Running setup.py egg_info for package from git+https://github.com/giampaolo/psutil.git
    warning: no previously-included files matching '*' found under directory 'docs/_build'
Installing collected packages: psutil
  Running setup.py install for psutil
    building 'psutil._psutil_linux' extension
    gcc -pthread -fno-strict-aliasing -O2 -g -pipe -Wall -Wp,-D_FORTIFY_SOURCE=2 -fexceptions -fstack-protector --param=ssp-buffer-size=4 -m64 -mtune=generic -D_GNU_SOURCE -fPIC -fwrapv -DNDEBUG -O2 -g -pipe -Wall -Wp,-D_FORTIFY_SOURCE=2 -fexceptions -fstack-protector --param=ssp-buffer-size=4 -m64 -mtune=generic -D_GNU_SOURCE -fPIC -fwrapv -fPIC -DPSUTIL_VERSION=300 -I/usr/include/python2.6 -c psutil/_psutil_linux.c -o build/temp.linux-x86_64-2.6/psutil/_psutil_linux.o
    psutil/_psutil_linux.c: In function ‘psutil_proc_cpu_affinity_set’:
    psutil/_psutil_linux.c:420: warning: suggest explicit braces to avoid ambiguous ‘else’
    gcc -pthread -shared build/temp.linux-x86_64-2.6/psutil/_psutil_linux.o -L/usr/lib64 -lpython2.6 -o build/lib.linux-x86_64-2.6/psutil/_psutil_linux.so
    building 'psutil._psutil_posix' extension
    gcc -pthread -fno-strict-aliasing -O2 -g -pipe -Wall -Wp,-D_FORTIFY_SOURCE=2 -fexceptions -fstack-protector --param=ssp-buffer-size=4 -m64 -mtune=generic -D_GNU_SOURCE -fPIC -fwrapv -DNDEBUG -O2 -g -pipe -Wall -Wp,-D_FORTIFY_SOURCE=2 -fexceptions -fstack-protector --param=ssp-buffer-size=4 -m64 -mtune=generic -D_GNU_SOURCE -fPIC -fwrapv -fPIC -I/usr/include/python2.6 -c psutil/_psutil_posix.c -o build/temp.linux-x86_64-2.6/psutil/_psutil_posix.o
    gcc -pthread -shared build/temp.linux-x86_64-2.6/psutil/_psutil_posix.o -L/usr/lib64 -lpython2.6 -o build/lib.linux-x86_64-2.6/psutil/_psutil_posix.so
    warning: no previously-included files matching '*' found under directory 'docs/_build'
Successfully installed psutil
Cleaning up...
```
```
wjia@wjia-ld1 ~/git/salt (git)-[psutil_compat] % python
Python 2.6.6 (r266:84292, Oct 12 2012, 14:23:48)
[GCC 4.4.6 20120305 (Red Hat 4.4.6-4)] on linux2
Type "help", "copyright", "credits" or "license" for more information.
>>> from psutil import *
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
AttributeError: 'module' object has no attribute 'phymem_buffers'
>>>
```

Grepping the code shows there is indeed no `phymem_buffers`.
```
wjia@wjia-ld1 ~/git/salt (git)-[psutil_compat] % grep -r phymem .v/lib/python2.6/site-packages/psutil/*
.v/lib/python2.6/site-packages/psutil/__init__.py:        total_phymem = _TOTAL_PHYMEM or virtual_memory().total
.v/lib/python2.6/site-packages/psutil/__init__.py:            return (rss / float(total_phymem)) * 100
Binary file .v/lib/python2.6/site-packages/psutil/__init__.pyc matches
.v/lib/python2.6/site-packages/psutil/_pslinux.py:    "phymem_buffers", "cached_phymem"]
Binary file .v/lib/python2.6/site-packages/psutil/_pslinux.pyc matches
```

 Looks like it was deprecated in 0.6.0, and should probably be removed from `_pslinux.py`'s `__extra__all__`, so it doesn't get added to `__all__`.

--- a/psutil/_pslinux.py
+++ b/psutil/_pslinux.py
@@ -40,10 +40,7 @@ __extra__all__ = [
     # connection status constants
     "CONN_ESTABLISHED", "CONN_SYN_SENT", "CONN_SYN_RECV", "CONN_FIN_WAIT1",
     "CONN_FIN_WAIT2", "CONN_TIME_WAIT", "CONN_CLOSE", "CONN_CLOSE_WAIT",
-    "CONN_LAST_ACK", "CONN_LISTEN", "CONN_CLOSING",
-    # other
-    "phymem_buffers", "cached_phymem"]
-
+    "CONN_LAST_ACK", "CONN_LISTEN", "CONN_CLOSING", ]
 
 # --- constants
 

